libs/core/langchain_core/runnables/base.py

```diff
@@ -438,12 +438,63 @@ def pipe(
         *others: Union[Runnable[Any, Other], Callable[[Any], Other]],
         name: Optional[str] = None,
     ) -> RunnableSerializable[Input, Other]:
-        """Compose this runnable with another object to create a RunnableSequence."""
+        """Compose this runnable with another object to create a RunnableSequence.
```

Comment:
Perhaps a simpler example would suffice?
pipe() is just another way of doing prompt | model | output_parser as prompt.pipe(model, output_parser)

So something like
```
from langchain_core.prompts import PromptTemplate
from langchain_community.llms.fake import FakeStreamingListLLM
from langchain_core.output_parsers import StrOutputParser

prompt = PromptTemplate.from_template("This is a prompt")
model = FakeStreamingListLLM(responses = ["I'm a textbot"])
output_parser = StrOutputParser()

chain = prompt.pipe(model, output_parser)
print(chain.output_schema.schema())
```

I'm not sure how adding `RunnablePassthrough` helps here.

@liugddx Regardless, thanks for taking up this activity. I think I got a grasp of the Runnable methods by looking over your pull requests :)